learn/guides/organize-ballerina-code/package-references.md

```diff
@@ -50,67 +52,67 @@ export = ["winery", "winery.model"]
 observabilityIncluded = true
 ```
 
-The `[package]` table contains the meta information about the package.
-
 ### The `org` field
 
 The organization is a logical name used for grouping modules together under a common namespace within a repository. Building a library package with `bal build -c` and pushing a library package into a repository will fail without an organization name.
 
-Organization names can only contain alphanumerics, underscore, and the maximum length is 256 characters.
+>**Info:** Organization names can only contain alphanumerics, underscore, and the maximum length is 256 characters.
 
-When you run the `bal new` command, the organization name by default will be set to the user name of your machine. You can choose to update the `Ballerina.toml` file to amend the organization name appropriately.
+When you run the `bal new` command, the organization name will be set to the user name of your machine by default. You can choose to update the `Ballerina.toml` file to amend the organization name appropriately.
 
 As described in [Organizations](/learn/publish-packages-to-ballerina-central/#organizations), the restrictions for the organization name of a package when publishing to [Ballerina Central](https://central.ballerina.io/) should also be considered before choosing an organization name.
 
-
 ### The `name` field
 
 The package name is an identifier used to refer to the package.
 
-The name can only contain alphanumerics, underscore, period, and the maximum length is 256 characters.
+>**Info:** The name can only contain alphanumerics, underscore, period, and the maximum length is 256 characters.
 
-If the package name is not provided in the `Ballerina.toml` file, then the current directory name is set as the package name. If there are any characters in the directory name mismatching the allowed regex, these will be replaced with the `_` character.
+If the package name is not provided in the `Ballerina.toml` file, then, the current directory name is set as the package name. If there are any characters in the directory name mismatching the allowed regex, these will be replaced with the `_` character.
 
 #### Hierarchical package names
 
-When a package provides multiple functionalities, it is better to split it into multiple packages instead.  For scenarios like this, you can give a hierarchical name to the package.
+When a package provides multiple functionalities, it is better to split it into multiple packages.  For scenarios like this, you can give a hierarchical name to the package.
 
 For example, if you need to provide a set of APIs to communicate with AWS, you can choose to support APIs to AWS services using multiple packages such as `aws.s3`, `aws.sqs`, `aws.rds`, etc.
 
-#### The split module condition
+#### The `split module` condition
```

Comment:
Since this is not a code or a file name, do we need a back-tick here?

Reply:
Removed the backticks.